css/css-text/text-align/reference/text-align-end-ref-016.html

```diff
@@ -1,20 +1,21 @@
 <!DOCTYPE html>
-<html  lang="en" >
+<html lang="en">
 <head>
 <meta charset="utf-8">
 <title>text-align: end, pre, dir=auto</title>
 <link rel='author' title='Richard Ishida' href='mailto:ishida@w3.org'>
-<link rel="stylesheet" type="text/css" href="/fonts/ahem.css" />
```

Comment:
If you're going to remove / at element ends, you may want to also remove type="text/css" (since the latter produces warnings in the html validator).